DEVELOPMENT_GUIDE.md

```diff
@@ -116,30 +95,116 @@ This step is optional and will use the specified version of
 aws-sam-transformer from PyPi by default.
 
 
-``cd ~/projects (cd into the directory where you usually place projects)``
+```sh
+$ cd ~/projects  # cd into the directory where you usually place projects
+# Using SSH
+$ git clone git@github.com:aws/serverless-application-model.git
+# Using HTTPS
+$ git clone https://github.com/aws/serverless-application-model.git
```

Comment:
I think one git clone with comment is enough. We already explained two ways of doing git clone and we can just pick one here for example.